packages/wallets/alpha/src/adapter.ts

```diff
@@ -185,13 +186,13 @@ export class AlphaWalletAdapter extends BaseMessageSignerWalletAdapter {
         }
     }
 
-    async signTransaction(transaction: Transaction): Promise<Transaction> {
+    async signTransaction<T extends Transaction>(transaction: T): Promise<T> {
```

Comment:
We know that transaction will only ever be `Transaction` now because `supportedTransactionVersions` is null, but we still have to implement it this way because the interface declares it like this. All what this means is that if a subtype of `Transaction` was passed to `signTransaction`, the wallet _should_ return the same subtype. In practice, this doesn't matter because there are no subclasses of `Transaction` in web3.js to worry about.